src/components/forms/AddressInput/index.tsx

```diff
@@ -4,12 +4,26 @@ import { OnChange } from 'react-final-form-listeners'
 
 import TextField from 'src/components/forms/TextField'
 import { composeValidators, mustBeEthereumAddress, required } from 'src/components/forms/validator'
+import { trimSpaces } from 'src/utils/strings'
 import { getAddressFromENS } from 'src/logic/wallets/getWeb3'
 import { isValidEnsName } from 'src/logic/wallets/ethAddresses'
 
 // an idea for second field was taken from here
 // https://github.com/final-form/react-final-form-listeners/blob/master/src/OnBlur.js
 
+export interface AddressInputProps {
+  fieldMutator: (address: string) => void
+  name?: string
+  text?: string
+  placeholder?: string
+  inputAdornment?: { endAdornment: React.ReactElement } | undefined
+  testId: string
+  validators?: Array<(data: string) => string>
```

Comment:
You can use `Validator` type from https://github.com/gnosis/safe-react/blob/development/src/components/forms/validator.ts#L10

Reply:
I'm having difficulties integrating it. This type doesn't fit 100% with current implementation. I'm having issues to use it with an empty array

Reply:
Found the issue why it wasn't working. I was using it wrong. Now it's working